src/controllers/networks/networks.ts

```diff
@@ -26,6 +28,8 @@ const STATUS_WRAPPED_METHODS = {
  * for adding, updating, and removing networks.
  */
 export class NetworksController extends EventEmitter {
+  defaultNetworksMode: 'mainnet' | 'testnet' = 'mainnet'
```

Comment:
Please add a few notes with context why this is needed (basically, parts of the PR description can go here).